LibreNMS/Snmptrap/Handlers/AdvaAccThresholdCrossingAlert.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * AdvaAccThresholdCrossingAlert.php
+ *
+ * -Description-
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Adva Threshold Exceeded Alarms.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2018 KanREN, Inc
+ * @author     Heath Barnhart <hbarnhart@kanren.net> & Neil Kahle <nkahle@kanren.net>
+ */
+
+namespace LibreNMS\Snmptrap\Handlers;
+
+use App\Models\Device;
+use LibreNMS\Interfaces\SnmptrapHandler;
+use LibreNMS\Snmptrap\Trap;
+
+class AdvaAccThresholdCrossingAlert implements SnmptrapHandler
+{
+    /**
+     * Handle snmptrap.
+     * Data is pre-parsed and delivered as a Trap.
+     *
+     * @param Device $device
+     * @param Trap $trap
+     * @return void
+     */
+    public function handle(Device $device, Trap $trap)
+    {
+        $raw = $trap->getRaw();
+        Log::event($raw, $device->device_id, 'trap', 2);
```

Comment:
Dumping the entire trap into the eventlog in addition to the event below might be too much noise.  (could this be debug code?)